shell.py

```diff
@@ -486,27 +486,28 @@ def setup_infrastructure(n_parties, chain='dev', regions=use_regions(), instance
 
 
 def setup_nodes(n_parties, chain='dev', regions=use_regions(), instance_type='t2.micro', volume_size=8, tag='dev',
-                **chain_flags):
+                node_flags=None, chain_flags=None):
```

Comment:
instead of None why not just dict()

Reply:
mutable default argument is not a good idea: https://docs.python-guide.org/writing/gotchas/

Reply:
I think the "pythonic" way to have mutable default arguments is to do:
```
def func(....., dict_arg=None);
     dict_arg = dict_arg or dict();
```
i.e. to do what you did, but once and on top of the function body. I don't think such a change is needed here, just dropping my 3 grosze ;)